app/src/main/java/org/schabi/newpipe/about/LicenseFragmentHelper.java

```diff
@@ -26,8 +28,8 @@ public LicenseFragmentHelper(@Nullable final Activity activity) {
         weakReference = new WeakReference<>(activity);
     }
 
-    private static String getFinishString(final Activity activity) {
-        return activity.getApplicationContext().getResources().getString(R.string.finish);
+    private static String getFinishString(final Context context) {
+        return context.getString(R.string.finish);
```

Comment:
Nothing to do with dependency updates, but what's the purpose of this function? `context.getString(R.string.finish)` is already explicative, readable and not long, so why encapsulate it in a function?

Reply:
I was wondering that as well… I'll fix it later today